src/main/charts/confluence/templates/ingress.yaml

```diff
@@ -37,7 +37,7 @@ spec:
                 port:
                   number: {{ $.Values.confluence.service.port }}
           {{ if .Values.synchrony.enabled }}
-          - path: {{ trimSuffix "/" (include "confluence.ingressPath" .) }}/synchrony"
+          - path: {{ trimSuffix "/" (include "confluence.ingressPath" .) }}/synchrony
```

Comment:
This is the bugfix for ingress typo that has been addressed in #281

Reply:
Shall we merge main into this branch in this case?